src/sentry/tasks/integrations/migrate_ignored_fields.py

```diff
@@ -0,0 +1,38 @@
+from typing import Any
+
+from sentry.models import OrganizationIntegration
+from sentry.tasks.base import instrumented_task
+from sentry.tasks.integrations import logger
+
+
+@instrumented_task(
+    name="sentry.tasks.integrations.migrate_ignored_fields",
+    queue="integrations",
+    default_retry_delay=60 * 5,
+    max_retries=5,
+)
+def migrate_ignored_fields(
```

Comment:
You can set ignored fields in the plugin and in the integration so this migrates that setting to the integration. If they are set in both, this combines the two, if it's just set in the plugin and not the integration, it adds the plugin settings to the integration.